### ISSUE TITLE
Compatibility certification request for Eclipse Mojarra 3.0.0

### ISSUE THREAD

**Issue by arjantijms** [eclipse-ee4j/mojarra#4742]
- [x] Organization Name ("Organization") and, if applicable, URL\
  Eclipse Foundation
- [x] Product Name, Version and download URL (if applicable)\
  [Mojarra 3.0.0](https://eclipse-ee4j.github.io/mojarra)
- [x] Specification Name, Version and download URL\
   [Jakarta Server Faces 3.0](https://jakarta.ee/specifications/faces/3.0/)
- [x] TCK Version, digital SHA-256 fingerprint and download URL\
  [Jakarta Server Faces TCK 3.0.0](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted/jakarta-faces-tck-3.0.0.zip), SHA-256: bba2c7ea38babed1e2d94236c4391e01bf3bfed45934dc6505365126644084b0
- [x] Public URL of TCK Results Summary\
  [TCK results summary](https://eclipse-ee4j.github.io/mojarra/certifications/jakarta-faces/3.0/TCK-Results)
- [x] Any Additional Specification Certification Requirements\
  None
- [x] Java runtime used to run the implementation\
  Oracle JDK 1.8.0_202
- [x] Summary of the information for the certification environment, operating system, cloud, ...\
  Alpine Linux v3.12
- [x] By checking this box I acknowledge that the Organization I represent accepts the terms of the [EFTL](https://www.eclipse.org/legal/tck.php).
- [x] By checking this box I attest that all TCK requirements have been met, including any compatibility rules.

**Comment by arjantijms** [comment]
Note: the SHA `bba2c7ea38babed1e2d94236c4391e01bf3bfed45934dc6505365126644084b0` is valid as-of 27-09-2020, but is likely to be changed still.

**Comment by Pandrex247** [comment]
The link for the TCK results seems to be a dead link?

**Comment by arjantijms** [comment]
What was I thinking. I put `mojarra` in the link instead of `glassfish`.

**Comment by arjantijms** [comment]
SHA `67e7acb555ac2c8955b4157118ef86a5fbb57fb9a65e3d0d75992ec83004204e` valid as of 03-10-2020

**Comment by edbratt** [comment]
This issue should be filed with the Faces API repository. The TCK Summary should be with the Mojarra Implementation, but the certification request needs to be moved/copied over to the faces-api repository.

**Comment by arjantijms** [comment]
@edbratt 

You're right, last time around it was moved as well: https://github.com/eclipse-ee4j/mojarra/issues/4620
